ledgers-user-management/ledgers-user-management-service-api/src/main/java/de/adorsys/ledgers/um/api/service/UserService.java

```diff
@@ -7,11 +7,18 @@
 
 //NOPMD
 
-import de.adorsys.ledgers.um.api.domain.*;
-import org.springframework.data.domain.*;
-
-import java.time.*;
-import java.util.*;
+import de.adorsys.ledgers.um.api.domain.UserBO;
+import de.adorsys.ledgers.um.api.domain.ScaUserDataBO;
+import de.adorsys.ledgers.um.api.domain.AccountAccessBO;
+import de.adorsys.ledgers.um.api.domain.AisConsentBO;
+import de.adorsys.ledgers.um.api.domain.UserExtendedBO;
+import de.adorsys.ledgers.um.api.domain.UserRoleBO;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
 
 public interface UserService {
```

Comment:
These is a different module, it should not be here.

looks like you deleted things..

Please fix these.

Reply:
This was necessary to correct a failing pmd check.